pkgs/by-name/as/astrometry-net/package.nix

```
@@ -0,0 +1,32 @@
+{ lib, stdenv, fetchFromGitHub, python3, cairo, netpbm, libpng, libjpeg, zlib, bzip2, swig, cfitsio, pkg-config, gsl }:
+
+stdenv.mkDerivation (FinalAttr: {
+  pname = "astrometry.net";
+  version = "0.94";
+  version_date = "2023-05-01";
+
+  src = fetchFromGitHub {
+    owner = "dstndstn";
+    repo = FinalAttr.pname;
+    rev = FinalAttr.version;
```

```suggestion
  pname = "astrometry.net";
  version = "0.94";

  src = fetchFromGitHub {
    owner = "dstndstn";
    repo = "astrometry.net";
    rev = finalAttrs.version;
```

Shouldn't the package name be without .net?